src/symbols.c

```diff
@@ -1927,15 +1927,23 @@ static void show_goto_popup(GeanyDocument *doc, GPtrArray *tags, gboolean have_b
 	GdkEventButton *button_event = NULL;
 	TMTag *tmtag;
 	guint i;
-
+	gchar **short_names, **file_names, **p;
 	menu = gtk_menu_new();
 
+	/* If popup would show multiple files presend a smart file list that allows
```

Comment:
`presend` sb `present`?

Reply:
@elextr seems to indeed have spotted a small typo :)